src/main/java/edu/cornell/kfs/paymentworks/service/impl/PaymentWorksAchServiceImpl.java

```diff
@@ -0,0 +1,298 @@
+/*
+ * The Kuali Financial System, a comprehensive financial management system for higher education.
+ *
+ * Copyright 2005-2014 The Kuali Foundation
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package edu.cornell.kfs.paymentworks.service.impl;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.pdp.PdpConstants;
+import org.kuali.kfs.pdp.businessobject.PayeeACHAccount;
+import org.kuali.kfs.sys.KFSKeyConstants;
+import org.kuali.rice.core.api.datetime.DateTimeService;
+import org.kuali.kfs.krad.exception.ValidationException;
+import org.kuali.kfs.krad.util.GlobalVariables;
+import org.kuali.kfs.krad.util.ObjectUtils;
+
+import edu.cornell.kfs.paymentworks.PaymentWorksConstants;
+import edu.cornell.kfs.paymentworks.batch.report.AchUpdateSummary;
+import edu.cornell.kfs.paymentworks.batch.report.AchUpdateSummaryLine;
+import edu.cornell.kfs.paymentworks.businessobject.PaymentWorksVendor;
+import edu.cornell.kfs.paymentworks.service.PaymentWorksAchService;
+import edu.cornell.kfs.paymentworks.service.PaymentWorksKfsService;
+import edu.cornell.kfs.paymentworks.service.PaymentWorksUtilityService;
+import edu.cornell.kfs.paymentworks.service.PaymentWorksVendorService;
+import edu.cornell.kfs.paymentworks.service.PaymentWorksWebService;
+import edu.cornell.kfs.paymentworks.xmlObjects.PaymentWorksUpdateVendorStatus;
+import edu.cornell.kfs.paymentworks.xmlObjects.PaymentWorksVendorUpdatesDTO;
+import edu.cornell.kfs.sys.service.ReportWriterService;
+
+public class PaymentWorksAchServiceImpl implements PaymentWorksAchService {
+	private static final String ROW_FORMAT_EDIT_SUMMARY = "%-15s %-30s %-20s";
+	private static final String ROW_FORMAT_REJECT_SUMMARY = ROW_FORMAT_EDIT_SUMMARY + " %s";
+	
+	private class HEADER_ARGUMENTS {
```

Comment:
Another potential area where it might be good to rename the nested class for consistency.